lib/ex_webrtc/dtls_transport.ex

```diff
@@ -195,16 +209,16 @@ defmodule ExWebRTC.DTLSTransport do
         state = %{state | buffered_packets: packets}
         update_dtls_state(state, :connecting)
 
-      {:handshake_finished, _, remote_keying_material, profile, packets} ->
+      {:handshake_finished, lkm, rkm, profile, packets} ->
         Logger.debug("DTLS handshake finished")
         ICEAgent.send_data(state.ice_agent, packets)
         # TODO: validate fingerprint
-        state = setup_srtp(state, remote_keying_material, profile)
+        state = setup_srtp(state, lkm, rkm, profile)
```

Comment:
we could make setup_srtp return :ok

Reply:
why tho? Do you mean that `setup_srtp` could fail when provided with these keying materials or profile?